tests/test_query.py

```diff
@@ -1240,3 +1240,12 @@ def test_query_datetimes_second(self):
     def test_empty_queryset_bool_converts_to_false(self):
         qs = MockSet()
         assert not bool(qs)
+
+    def test_empty_queryset_filter(self):
+        car1 = Car(id=101)
+        car2 = Car(id=102)
+
+        mockset = MockSet(car1, car2)
+        self.assertEqual(mockset.count(), 2)
+        self.assertEqual(mockset.filter(Q()).count(), 2)
+
```

Comment:
flake8 lint error "blank line at end of file" :)

https://travis-ci.org/github/stphivos/django-mock-queries/jobs/754837922